root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlSvr.cs

```diff
@@ -114,6 +114,11 @@ public SqlConnection DamSqlConnection
                 // コネクションを戻す
                 return _cnn;
             }
+            set
+            {
+                //set the connection
+                _cnn = value;
+            }
```

Comment:
@daisukenishino 

>I think that is the trade-off of the standardization and flexibility.

If so, for data dependent routing, how about creating new dam that inherits DamSqlSvr?